File: Code/Python/LAB03_03.py
# Function to calculate the sum of numbers from 1 to 100 using for loop
def sum_using_for():
    total = 0
    for i in range(1, 101):
        total += i
    return total

# Function to calculate the sum of numbers from 1 to 100 using while loop
def sum_using_while():
    total = 0
    i = 1
    while i <= 100:
        total += i
        i += 1
    return total

# Function to calculate the sum of numbers from 1 to 100 using do-while loop
def sum_using_do_while():
    total = 0
    i = 1
    while True:
        total += i
        i += 1
        if i > 100:
            break
    return total

def main():
    print("Sum using for loop:", sum_using_for())
    print("Sum using while loop:", sum_using_while())
    print("Sum using do-while loop:", sum_using_do_while())

if __name__ == "__main__":
    main()

File: Code/Cplus/LAB03_03/LAB03_03.cpp
#include <iostream>
using namespace std;

// Function to calculate the sum of numbers from 1 to 100 using for loop
int sumUsingFor() {
    int sum = 0;
    for (int i = 1; i <= 100; i++) {
        sum += i;
    }
    return sum;
}

// Function to calculate the sum of numbers from 1 to 100 using while loop
int sumUsingWhile() {
    int sum = 0, i = 1;
    while (i <= 100) {
        sum += i;
        i++;
    }
    return sum;
}

// Function to calculate the sum of numbers from 1 to 100 using do-while loop
int sumUsingDoWhile() {
    int sum = 0, i = 1;
    do {
        sum += i;
        i++;
    } while (i <= 100);
    return sum;
}

int main() {
    cout << "Sum using for loop: " << sumUsingFor() << endl;
    cout << "Sum using while loop: " << sumUsingWhile() << endl;
    cout << "Sum using do-while loop: " << sumUsingDoWhile() << endl;
    return 0;
}
